xdebug_gc_stats.c

```diff
@@ -0,0 +1,157 @@
+/*
+   +----------------------------------------------------------------------+
+   | Xdebug																  |
```

Comment:
Something seems very much not aligned here? But that could be GitHub rendering it stupidly.